src/pykx/pandas_api/pandas_meta.py

```diff
@@ -210,6 +244,37 @@ def abs(self, numeric_only=False):
             tab = _get_numeric_only_subtable(self)
         return q.abs(tab)
 
+    @api_return
+    def round(self, decimals: Union[int, Dict[str, int]] = 0):
+        tab = self
+        if 'Keyed' in str(type(tab)):
+            tab = q('{(keys x) _ 0!x}', tab)
```

Comment:
In this [pr](https://github.com/KxSystems/pykx/pull/16), it was said that it was better to use `q.value(tab)`

Reply:
Done